pz_be_services/routers/v1/user_router.py

```
@@ -194,6 +193,110 @@ async def github_callback( code: str = None, db: Session = Depends(get_db)):
+            raise HTTPException(
+                status_code=400, 
+                detail="Unable to create user account with provided Google data"
+            )
+        except HTTPException as e:
+            raise e
+        except Exception as e:
+            logger.exception(str(e))
+            raise HTTPException(
+                status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+                detail=str(e),
+            )
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)